tasks/test.py

```diff
@@ -21,20 +21,18 @@ def lint(ctx):
         unit="run unit tests (pytest)",
         installed="like unit, but prefer using installed package, "
         + "and do not allow the use of internet.",
-        style="run style tests (flake8)",
+        style="run style tests (flake8 and black)",
```

Comment:
I'm a little confused by our dependency on black. Are we enforcing it somewhere outside of the tests? Is it required to use black-formatted code for commits? If so, I think we should mention it in the contribution guidelines somewhere.

Reply:
Well, we do mention in the readme to run `invoke format`, which runs black. I'll tweak the text to mention Black.